src/Idris/Docs.hs

```diff
@@ -87,15 +96,21 @@ pprintFD ist (FD n doc args ty f)
                showArgs args ((n, True):bnd)
           showArgs ((n, _, _, _):args) bnd = showArgs args ((n, True):bnd)
           showArgs []                  _ = []
+          getTotality = map (text . show) $ lookupTotal n (tt_ctxt ist)
 
+pprintFDWithTotality :: IState -> FunDoc -> Doc OutputAnnotation
```

Comment:
This is reasonable enough, but if we add one more parameter, then we should make an options structure rather than having lots of parameters.